nixio/block.py

```diff
@@ -290,6 +290,54 @@ def find_sources(self, filtr=lambda _: True, limit=None):
             limit = maxint
         return finders._find_sources(self, filtr, limit)
 
+    def pprint(self, indent=2, max_row=50, max_length=80, max_depth=2):
+        spaces = " " * (0 * indent)
+        p = "{} {} {}".format(spaces,  self.name, self.type)
+        print(p)
+        for grp in self.groups:
```

Comment:
Might be nice to print some indication that objects under `Group` are links.  Maybe something like an asterisk.

For instance:
```
Block: {name = Block 0, type = nix.Block, id = 4214ebeb-712e-4c64-8866-b5baf67277f4}
    Group: {name = Group 0-0, type = nix.Group, id = f5eaf3f3-2bf1-44c8-afdc-254cfec342df}
      DataArray: {name = DA0:0, type = nix.DataArray, id = 41755270-fffa-47e5-b002-23f852f2e750}
      Shape: (10, 1) Unit:None
        SampledDimension: {index = 1}
      DataArray: {name = DA0:1, type = nix.DataArray, id = 9bc6935d-976a-45e3-b915-2007bb66d486}
      Shape: (10, 1) Unit:None
        SampledDimension: {index = 1}
      DataArray: {name = DA0:2, type = nix.DataArray, id = 006c7201-d418-4646-b9a0-7dfc761e563a}
      Shape: (10, 1) Unit:None
        SampledDimension: {index = 1}
    DataArray: {name = DA0:0, type = nix.DataArray, id = 41755270-fffa-47e5-b002-23f852f2e750}
    Shape: (10, 1) Unit:None
      SampledDimension: {index = 1}
    DataArray: {name = DA0:1, type = nix.DataArray, id = 9bc6935d-976a-45e3-b915-2007bb66d486}
    Shape: (10, 1) Unit:None
      SampledDimension: {index = 1}
    DataArray: {name = DA0:2, type = nix.DataArray, id = 006c7201-d418-4646-b9a0-7dfc761e563a}
    Shape: (10, 1) Unit:None
      SampledDimension: {index = 1}
```

Would become
```
Block: {name = Block 0, type = nix.Block, id = 4214ebeb-712e-4c64-8866-b5baf67277f4}
    Group: {name = Group 0-0, type = nix.Group, id = f5eaf3f3-2bf1-44c8-afdc-254cfec342df}
      *DataArray: {name = DA0:0, type = nix.DataArray, id = 41755270-fffa-47e5-b002-23f852f2e750}
       Shape: (10, 1) Unit:None
        SampledDimension: {index = 1}
      *DataArray: {name = DA0:1, type = nix.DataArray, id = 9bc6935d-976a-45e3-b915-2007bb66d486}
       Shape: (10, 1) Unit:None
        SampledDimension: {index = 1}
      *DataArray: {name = DA0:2, type = nix.DataArray, id = 006c7201-d418-4646-b9a0-7dfc761e563a}
       Shape: (10, 1) Unit:None
        SampledDimension: {index = 1}
    DataArray: {name = DA0:0, type = nix.DataArray, id = 41755270-fffa-47e5-b002-23f852f2e750}
    Shape: (10, 1) Unit:None
      SampledDimension: {index = 1}
    DataArray: {name = DA0:1, type = nix.DataArray, id = 9bc6935d-976a-45e3-b915-2007bb66d486}
    Shape: (10, 1) Unit:None
      SampledDimension: {index = 1}
    DataArray: {name = DA0:2, type = nix.DataArray, id = 006c7201-d418-4646-b9a0-7dfc761e563a}
    Shape: (10, 1) Unit:None
      SampledDimension: {index = 1}
```
or similar.